web-ui/src/main/java/org/archcnl/ui/MainPresenter.java

```diff
@@ -100,9 +98,6 @@ public MainPresenter() throws ConceptDoesNotExistException {
                 event -> event.handleEvent(conceptManager.getInputConcepts()));
         inputPresenter.addListener(
                 ConceptSelectedEvent.class, event -> event.handleEvent(conceptManager));
-        inputPresenter.addListener(DeleteRuleButtonPressedEvent.class, this::handleEvent);
-        inputPresenter.addListener(DeleteConceptRequestedEvent.class, this::handleEvent);
-        inputPresenter.addListener(DeleteRelationRequestedEvent.class, this::handleEvent);
```

Comment:
These three listeners need to be added back. They were recently introduced in [PR 272](https://github.com/Mari-Wie/ArchCNL/pull/272).